themes/digital.gov/layouts/shortcodes/youtube.html

```diff
@@ -1,9 +1,29 @@
+{{/* Embeds a youtube player into the page
+  Use the share link on the youtube.com to get the id and start time
+
+  @param {string} id - the required youtube video id
+  @params {number} [start_time] - the optional start time in seconds for the video
+  @params {string} title - The required title of the video for screen reader accessibility
+
+  @example
+  {{< youtube id="Hvo7WVKvPGU" start_time"101" title="Youtube video title goes here"
+>}}
+
+*/}}
+{{ $id := (.Get "id") }}
+{{ $start_time := (.Get "start_time") }}
+{{ $title := (.Get "title") }}
+
+
 <div
   class="video"
   style="position: relative; padding-bottom: 56.25%; padding-top: 30px; height: 0; overflow: hidden;"
 >
   <iframe
-    src="//www.youtube.com/embed/{{ index .Params 0 }}"
+    src="https://www.youtube.com/embed/{{ $id }}?start={{ if $start_time }}
```

Comment:
Does the `?start=` still appear if you don't enter a value for `$start_time`?

Reply:
It does, moved the conditional to only show when `$start_time` exists. Disabled prettier formatting which was causing white spaces with the conditional check for the `$start_time`, see old formatting (02ce5c474a82ae28ec1e294fb4ad0e8a8209033d)

Reply:
What was the prettier issue? We should avoid ignoring wherever possible.